rpm/SOURCES/openresty.init

```diff
@@ -73,7 +73,7 @@ configtest() {
 }
 
 rh_status() {
-    status $nginx
+    status -p "$pidfile" $nginx
```

Comment:
~~Unluckily, I could not find any standard docs about init.d/function.~~
I found one in the `/usr/share/doc/initscripts-*/sysvinitfiles`.
The doc doesn't mention any arguments `status` accepts.
Seems it is a case for `no docs, but many people use it`.